pkg/converter/convert.go

```diff
@@ -44,6 +44,88 @@ const (
 	execAPIVersion  = "client.authentication.k8s.io/v1beta1"
 )
 
+type goody struct {
+	// 1. cfgApiserverID - argServerIDVal
+	// 2. cfgClientID - argClientIDVal
+	// 3. cfgEnvironment - argEnvironmentVal
+	// 4. cfgTenantID - argTenantIDVal
+	// 5. cfgConfigMode - cfgConfigModeVal
+
+	// exec formatted kubeconfig doesn't use --config-mode (1) nor apiserver-id (5)
+
+	argEnvironmentVal, argClientIDVal, argServerIDVal, argTenantIDVal, cfgConfigModeVal string
```

Comment:
now i'm really not getting the value of `goody`. it's nothing but a container of variables that are already scoped in the function, right? what else does it provide?